src/js/themes/base.js

```diff
@@ -378,6 +378,58 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         // fontWeight: undefined,
         // extend: undefined,
       },
+      // size: {
```

Comment:
This set of examples seems a bit long winded. I'm wondering if we can pull this down to just `\\ size: undefined,`, or `// size: {},` or `// size: { medium: { color: undefined, ... }, }`?

Reply:
Yes, we can prune it down then have docs/d.ts file to give more clarity on structure.